go/cmd/lemon-mint5/main.go

```
@@ -0,0 +1,157 @@
+			processChunk(data, bounds[worker], bounds[worker+1], &tables[worker])
+		}()
+	}
+	last := workers - 1
+	processChunk(data, bounds[last], bounds[last+1], &tables[last])
+	wait.Wait()
+
+	runtime.KeepAlive(data)
+	runtime.KeepAlive(tables)
+	return mergeAndFormat(tables)
+}
+func solution(inputPath string) []byte {
```

golangci-lint (2.12.2)

[error] func solution is unused

(unused)